docker/Dockerfile

```diff
@@ -1,32 +1,24 @@
-FROM formapro/nginx-php-fpm:latest-all-exts
+FROM formapro/nginx-php-fpm:7.3-latest-all-exts
 
 ## libs
 RUN set -x && \
     apt-get update && \
-    apt-get install -y --no-install-recommends --no-install-suggests wget curl openssl ca-certificates nano netcat php-dev php-redis php-pgsql git python
-
-RUN set -x && \
-    apt-get update && \
-    apt-get install -y --no-install-recommends --no-install-suggests php-dev librabbitmq-dev make  && \
-    mkdir -p $HOME/php-amqp && \
-    cd $HOME/php-amqp && \
-    git clone https://github.com/pdezwart/php-amqp.git . && git checkout v1.9.3 && \
-    phpize --clean && phpize && ./configure && make install
```

Comment:
from what i can see this is already installed in the formapro image and it is at `1.9.4`, if you want to move to the latest version `1.10.2` let me know, then i can add it in the `apt-get install` section

Reply:
Done :smile: